pkg/k8shandler/fluentd.go

```diff
@@ -215,6 +215,7 @@ func newFluentdPodSpec(logging *logging.ClusterLogging, elasticsearchAppName str
 	}
 
 	fluentdContainer.Env = []v1.EnvVar{
+		{Name: "NODE_NAME", ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{FieldPath: "spec.nodeName"}}},
```

Comment:
we'll need to change fluentd and rsyslog to consume this
do we need to add this? If I `oc rsh` into a pod and do `env`, I see there are environment variables related to the hostname - are there some we can already use?

Reply:
I have not looked at the list but typically 'hostname' is the name of the pod.  The link you posted to me previously seemed to make it obvious that we don't use the node hostname anywhere so if we don't need the value then we could simply remove adding it all together

Reply:
I'm looking at a recent 4.2 cluster.  It seems that all of our pods have a `HOSTNAME` env. var. added by Kubernetes?  OpenShift?
```
HOSTNAME=fluentd-xds7j
HOSTNAME=kibana-544698d578-qrltc
```
etc.
So I think we should just rely on the presence of the HOSTNAME env. var. in both fluentd and rsyslog, and get rid of the use of /etc/docker-hostname and /etc/hostname.  This means changes in origin-aggregated-logging/fluent-plugin-viaq_data_model and cluster-logging-operator.

Reply:
This hostname is the actual pod itself.  Wouldn't the nodename on which the fluent pod ran be useful in logging to understand on which node a given pod was running?  I don't have a strong opinion about keeping in the env var so I defer to if you believe we should simply remove it.

Reply:
ok, I see.  Yes, this is very useful, we should keep this - we'll have to change origin-aggregated-logging/fluent-plugin-viaq_data_model/cluster-logging-operator to use it